Patch-Source: https://github.com/aio-libs/pytest-aiohttp/pull/115
---
From 26b8c8c1b12769c5b94d4080e9ef83a50c1d9e28 Mon Sep 17 00:00:00 2001
From: Maxim Tulskiy <tulskijms@altlinux.org>
Date: Tue, 23 Sep 2025 11:58:47 +0300
Subject: [PATCH] tests: add pytest.ini to set
 asyncio_default_fixture_loop_scope

Recent versions of pytest-asyncio and pytest introduced
stricter handling of asynchronous fixtures. If the
'asyncio_default_fixture_loop_scope' option is unset, async fixtures may
emit DeprecationWarnings during pytest configuration. This can break tests
using pytester by preventing pytest from generating a terminal summary.

This prevents the warning and ensures consistent and stable test execution
for async tests.

Signed-off-by: Maxim Tulskiy <tulskijms@altlinux.org>
---
 pytest.ini | 2 ++
 1 file changed, 2 insertions(+)
 create mode 100644 pytest.ini

diff --git a/pytest.ini b/pytest.ini
new file mode 100644
index 0000000..0102b0a
--- /dev/null
+++ b/pytest.ini
@@ -0,0 +1,2 @@
+[pytest]
+asyncio_default_fixture_loop_scope = function
